pylint/config/find_default_config_files.py

```diff
@@ -17,7 +17,28 @@
     import tomli as tomllib
 
 RC_NAMES = (Path("pylintrc"), Path(".pylintrc"))
-CONFIG_NAMES = RC_NAMES + (Path("pyproject.toml"), Path("setup.cfg"))
+PYPROJECT_NAME = Path("pyproject.toml")
```

Comment:
```suggestion
PYPROJECT_NAME = "pyproject.toml"
```

Reply:
I missed that, it felt like we did not need a path in this particular case, but let's keep it that way.